sdk/compute/Microsoft.Azure.Management.Compute/src/Generated/Models/DedicatedHostGroup.cs

```diff
@@ -54,8 +54,8 @@ public DedicatedHostGroup()
         /// machines or virtual machine scale sets can be placed automatically
         /// on the dedicated host group. Automatic placement means resources
         /// are allocated on dedicated hosts, that are chosen by Azure, under
-        /// the dedicated host group. The value is defaulted to 'true' when not
-        /// provided. &lt;br&gt;&lt;br&gt;Minimum api-version:
+        /// the dedicated host group. The value is defaulted to 'false' when
```

Comment:
Is this a change in service behavior, or a correction to match service behavior in docs?

Reply:
This file has been generated by autorest tool using the last published swagger json (i.e. 30-06-2020).

Reply:
This is possibly a bug in Autorest. I have seen this pop up in a number of Microsoft docs and public facing docs.